panel/lxqtpanel.cpp

```diff
@@ -177,13 +177,17 @@ LxQtPanel::LxQtPanel(const QString &configGroup, QWidget *parent) :
     LxQtPanelApplication *app = reinterpret_cast<LxQtPanelApplication*>(qApp);
     mSettings = app->settings();
     readSettings();
-    mHidden = false;//overriding configuration to show it first time
-    QTimer::singleShot(PANEL_HIDE_FIRST_TIME, this, SLOT(hidePanel()));
     // the old position might be on a visible screen
     ensureVisible();
     loadPlugins();
 
     show();
+
+    // show it the first first time, despite setting
+    if (mHidable) {
```

Comment:
please, follow the coding style -> opening brace on new line (to have opening brace at the same position as the closing one)